src/AppBundle/Controller/BlogController.php

```diff
@@ -36,12 +36,18 @@ class BlogController extends Controller
     /**
      * @Route("/", name="blog_index")
      */
-    public function indexAction()
+    public function indexAction(Request $request)
     {
-        $em = $this->getDoctrine()->getManager();
-        $posts = $em->getRepository('AppBundle:Post')->findLatest();
```

Comment:
in this case the query doesn't need to know the current page, these parameter is handled by the paginator